Description: Cinnamon Settings User: Don't print passwords to stdout
when changing password and cinnamon-settings is run from terminal.
Author: Clement Lefebvre
Origin: upstream
--- a/files/usr/lib/cinnamon-settings/modules/cs_user.py
+++ b/files/usr/lib/cinnamon-settings/modules/cs_user.py
@@ -398,7 +398,6 @@
     def change_password(self):
         oldpass = self.current_password.get_text()
         newpass = self.new_password.get_text()
-        print "Changing %s to %s" % (oldpass, newpass)
         passwd = pexpect.spawn("/usr/bin/passwd")
         time.sleep(0.5)
         passwd.sendline(oldpass)
